src/app/knowledge-bases/knowledge-base-details/knowledge-base-details/knowledge-base-details.component.ts

```diff
@@ -328,4 +316,20 @@ export class KnowledgeBaseDetailsComponent implements OnInit, AfterViewInit, OnD
   pasteLookupList(projectId: string, baseId: string, value: string, split: string, remove: boolean) {
     this.knowledgeBaseApolloService.pasteTerm(projectId, baseId, value, split, remove).pipe(first()).subscribe();
   }
+
+  executeOption(value: string, term: any) {
+    switch(value) {
+      case 'Edit term': 
+        this.openTermEditor(true, term.id, term.value, term.comment);
+        break;
+      case 'Remove term':
+        this.deleteTerm(term.id);
+        break;
+      case 'Blacklist term':
+      case 'Whitelist term':
+        this.blackListTerm(term.id);
+        break;
```

Comment:
from the function names and the case this looks like a mistake. Maybe the function `this.blackListterm`... should be changed to `this.toggleBlacklistTerm` to reduce confusion. (I know that the backend also works with a toggle even though the name isn't representative of it but we need to start somewhere :) )

Reply:
Yes sure, changed